src/main/java/com/meilisearch/sdk/model/Details.java

```diff
@@ -1,15 +1,10 @@
-package com.meilisearch.sdk;
+package com.meilisearch.sdk.model;
 
 import java.util.Map;
 import lombok.Getter;
-import lombok.Setter;
 
 @Getter
-@Setter
 public class Details {
```

Comment:
I think we should take the time to find a better name for this class.

Reply:
Yes, this is the name of the field but we can change it for TaskDetails maybe?

Reply:
I liked :)